src/frontend/web_application/src/scenes/ContactList/components/ContactItem/index.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { PropTypes } from 'react';
+import { Link } from 'react-router';
+import ContactAvatarLetter from '../../../../components/ContactAvatarLetter';
+
+const ContactItem = ({ contact }) => (
+  <Link
+    to={`/contacts/${contact.contact_id}`}
+    className="s-contact-list__contact m-block-list__item-content m-block-list__item-content--link"
```

Comment:
that's strange `m-block-list__item-content` should not be here how is it defained on discussionList ?